src/containers/PipelineResources/PipelineResources.js

```diff
@@ -220,7 +220,7 @@ export /* istanbul ignore next */ class PipelineResources extends Component {
         ];
 
     return (
-      <>
+      <ListPageLayout title="PipelineResources" {...this.props}>
```

Comment:
Should the titles be externalised strings ? Probably not given theyre types but just to double check

Reply:
not for these since we're using the resource `kind` names from Kubernetes and don't translate them